Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -245,7 +245,6 @@ void updateLayersForInteractionRegions(const RemoteLayerTreeNode& node)
                 }
                 break;
             }
-            [layer insertSublayer:regionLayer.get() atIndex:insertionPoint];
```

Comment:
Noted in the commit message, but `foundInPosition` is always false here. So we take the branch on line 251 below which is the same line of code removed here